README.md

```diff
@@ -1,318 +1,194 @@
+for the documentation source.
 
-  The patched compiler is installed locally into your `$(pwd)/bin`
-  directory; check if `ocamlc.opt` and `ocamlopt.opt` are there, and then temporarily
-  add them into your `$(PATH)` (eg - `PATH=$(pwd)/bin:$PATH`).
+## How BuckleScript Works
 
-3. Build BuckleScript Compiler
+BuckleScript leverages the existing OCaml compiler and toolchain to produce JavaScript that closely
+resembles the original OCaml. This has several immediate advantages:
 
-  Assume that you have `ocamlopt.opt`(the one which we just built) in the `PATH`
+* name mangling is avoided
+* stack traces are preserved
+* OCaml modules are mapped to JavaScript modules
+* OCaml optimizations (e.g., constant folding, DCE, TCO) can be re-used
+* OCaml attributes allow fine control over generated JavaScript
 
-  ```sh
-  cd ./jscomp
-  npm_package_name=bs-platform make world
-  ```
+These points make the integration of JavaScript with OCaml very clean and smooth. In this way,
+BuckleScript provides all the benefits of OCaml's excellent compiler and sophisticated type system
+alongside the rich cross-platform JavaScript ecosystem.
 
-  Now that you have a binary called `bsc` in the `jscomp/bin` directory,
-  put it in your `PATH`.
+## BuckleScript Examples
 
+Basic examples of using BuckleScript are provided below. More extensive examples are available at
+https://github.com/bloomberg/bucklescript-addons.
 
-  Note that by default, `bsc` will generate `commonjs` modules. You can
-  override this behavior to generate JS files for all three module systems 
-  by setting a variable:
+#### An HTTP Server
 
+This example creates a simple http server. The complete code is available
+[here](https://github.com/bloomberg/bucklescript-addons/tree/master/examples/node-http-server).
 
-  ```sh
-  BS_RELEASE_BUILD=1 npm_package_name=bs-platform make world
-  ```
+The attribute `[@bs]` used below is one of the OCaml attributes mentioned earlier. In this case the
+attribute controls whether the generated code uses the uncurried (JavaScript) or curried (OCaml)
+calling convention. By decorating functions and call-sites with `[@bs]` you can write OCaml in the
+usual curried style and have BuckleScript take care of the conversion automatically.
 
-4. Test
+##### Input:
 
-  Outside of the `bucklescript` directory, create a separate directory with a file called `hello.ml`:
-
-  ```sh
-  mkdir tmp
-  cd tmp
-  echo 'print_endline "hello world";;' >hello.ml
-  ```
-
-  Then compile it with `bsc`.
-  ```sh
-  bsc -I ../bucklescript/jscomp/runtime -I ../bucklescript/jscomp/stdlib -c hello.ml
-  ```
-
-  It should generate a file called `hello.js`, which can be executed with any JavaScript engine. In this example, we use Node.js:
-
-  ```sh
-  node hello.js
-  ```
-
-  If everything goes well, you will see `hello world` on your screen.
-
-
-## Windows support
-
-We plan to provide a Windows installer in the near future.
```

Comment:
Okay. What about changing it to a `Releases` section and including a link to the github repository releases? Or are you planning on hosting the binaries somewhere else?